css/amp.css

```diff
@@ -59,7 +59,7 @@ html.-amp-ios-embed {
 }
 
 /** Wrapper for iOS-Embed-Wrapper mode. */
-i-amp-html-wrapper {
+#i-amp-html-wrapper {
```

Comment:
There might be  a validation bug then, #i-amp-html-wrapper passes validation on https://validator.ampproject.org/